src/videojs.ads.js

```diff
@@ -246,6 +268,37 @@ var
       // just resume playback at the current time.
       player.play();
     }
+
+    i = suppressedTracks.list.length;
+    if (tracks.addTrack_) {
+      while (i--) {
+        track = suppressedTracks.list[i];
+        tracks.addTrack_(track);
```

Comment:
we've decided to forgot this altogether and just snapshot the modes and set them all to 'disabled'.